how-to/fine-tune-server-performance.rst

```diff
@@ -0,0 +1,80 @@
+.. fine-tune-performance:
+
+How to fine-tune your server's performance
+==========================================
+
+In the Divio Cloud Control Panel for your project, you can easily change the resources (instances,
+RAM, storage, transfer) allocated to your project to adjust for its needs.
+
+In most cases, having chosen suitable values, you won't need to make further changes.
+
+If your project has unusual demands or sees unusual traffic, you can also fine-tune some settings
+to match its needs even better.
+
+Optimising the settings for a high-traffic site isn't just a case of allocating more resources
+indiscriminately on the basis that "more is better".
+
+For example, a site that sees *large numbers* of requests that each represent a *low load* will
+benefit from being allowed to serve more concurrent requests, but that could make things worse for
+a site that sees a *smaller number* of requests each representing a *heavier load*.
+
+
+Aldryn Django settings
+----------------------
+
+The :ref:`Aldryn Django addon <aldryn-django>` is responsible for ensuring that uWSGI (the
+application gateway server) starts up with appropriate parameters.
+
+These parameters can be controlled by environment variables.
+
+..  note::
+
+    These environment variables represent uWSGI start-up settings. It's also possible to configure
+    :ref:`other uWSGI settings <uwsgi-configuration>`, but we don't recommend doing so unless you
+    are familiar with them already and understand their implications.
+
+
+Increase ``DJANGO_WEB_WORKERS``
```

Comment:
To answer your earlier question, these are all prefixed "DJANGO*" because they are passed into uWSGI via aldryn-django, and we (presumably) want to namespace these settings as runtime variables rather than uWSGI settings
https://github.com/aldryn/aldryn-django/blob/48201876836458178068a0e419839bd8a02199da/aldryn_django/cli.py#L137